helm/cluster-operator/templates/configmap.yaml

```diff
@@ -30,7 +30,7 @@ data:
           domain: '{{ .Values.Installation.V1.Registry.Domain }}'
       kubeconfig:
         resource:
-          namespace: 'giantswarm'
+          namespace: 'giantswarm'  // TODO replace this too?
```

Comment:
I wasn't sure whether to replace it as it's data and I'm not too sure what this `config.yml` is?

Reply:
I think we should leave it.
CC @giantswarm/team-batman

Reply:
Ack. Makes sense to me.

Reply:
Done.